common/vpcclient/models/snapshot.go

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2025 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package models ...
+package models
+
+import "time"
+
+// SnapshotList ...
+type SnapshotList struct {
+	First      *HReference `json:"first,omitempty"`
+	Next       *HReference `json:"next,omitempty"`
+	Snapshots  []*Snapshot `json:"snapshots"`
+	Limit      int         `json:"limit,omitempty"`
+	TotalCount int         `json:"total_count,omitempty"`
+}
+
+// LisSnapshotFilters ...
+type LisSnapshotFilters struct {
+	Name string `json:"name,omitempty"`
+}
+
+// Snapshot ...
+type Snapshot struct {
```

Comment:
we need to discuss on these structs parameters

Reply:
We had done initial discussion already and we had agreed on this. IF you have any more concerns we can discuss.

Reply:
changed the sequence and put in line

```
CreatedAt        *time.Time        `json:"created_at,omitempty"`
	CapturedAt       *time.Time        `json:"captured_at,omitempty"`
```